### ISSUE TITLE
Feature Request: Switch active preview to current markdown file

### ISSUE THREAD

**Issue by abhayghatpande** [shd101wyy/markdown-preview-enhanced#435]
When I have multiple previews open, and I switch to one of the markdown files, can the corresponding preview be made "active"? Currently, the preview does not switch when I switch the markdown file.

If someone wants to view another file's preview, it is always possible to manually switch the preview pane.

I think logically it follows that the preview should match the file being edited.

**Comment by shd101wyy** [comment]
Do you have any advice for the naming of this config? Does the one below make sense?    

![screen shot 2017-06-05 at 12 18 41 am](https://cloud.githubusercontent.com/assets/1908863/26771549/a135b846-4984-11e7-9bc8-ae7cd23e5a3c.png)
 
Thanks

**Comment by abhayghatpande** [comment]
How about "Active preview pane shows preview of markdown file being edited"?

Seriously though, I think this behaviour should be the default, and the option should be:

"Do not switch preview pane when switching markdown files"

Thank you anyway. I'm happy as long as both behaviours exist!

**Comment by shd101wyy** [comment]
Right now the behavior is the same as the markdown-preview package such if you edit the markdown file, then the preview will be active correspondingly. Does that fulfill your requirement?

**Comment by abhayghatpande** [comment]
I'm not really sure ... There are 2 cases.

1. When I use the keyboard shortcut to open the preview the first time, it does open / activate a new preview pane for the current markdown file.

2. Now I've 3 previews open. When I switch to another markdown file, the preview pane does not switch to the preview of that markdown file. It remains fixed to the last viewed preview. I've to manually  click on the preview pane for the markdown file that I've switched to. (Or use the shortcut to toggle the preview off and then on again.)

All I was saying is that as a markdown previewer, it makes sense to activate the preview corresponding to the markdown file that is "active". Then, if someone wants to use the preview of another file while editing their markdown, they can always click on the other preview to activate it.

**Comment by shd101wyy** [comment]
Understood

**Comment by abhayghatpande** [comment]
@shd101wyy as a corollary to this, when a markdown file is closed, it's corresponding preview pane should be closed too (if it was open).

**Comment by shd101wyy** [comment]
No, someone requires that the preview keeps open. So I think I will just leave it like that.

**Comment by abhayghatpande** [comment]
Ok. No chance of adding another config item? ;) :)

**Comment by shd101wyy** [comment]
Sure

**Comment by shd101wyy** [comment]
Done.

Cheers :)
